.github/workflows/assignproj.yml

```diff
@@ -0,0 +1,21 @@
+name: Auto Assign to Project
+
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    types: [opened]
```

Comment:
I am guessing we also want to run this on reopened issues and PRs, correct?

Reply:
Hi Rachel, once an issue ticket is assigned a projects the first time, we don't have to reassigned it on a reopened.  Is there a different use case that I'm not following.  Moving this conversation to slack.

Reply:
I was thinking about the (admittedly not all that likely) possibility that an issue or pull request created before this PR was merged would be reopened and left without a project assigned. Per our Slack conversation, @angelatan2 will run a query separately to make sure there are no old ones without a project name.